rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -448,3 +448,10 @@ maas_excluded_checks: []
 # This is necessary until LP #1537117 is implemented
 openrc_os_endpoint_type: internalURL
 openrc_insecure: "{{ (keystone_service_adminuri_insecure | bool or keystone_service_internaluri_insecure | bool) | default(false) }}"
+# Name of the virtual env to deploy into (maas)
+maas_venv_tag: untagged
+maas_venv: "/openstack/venvs/maas-{{ maas_venv_tag }}"
+maas_venv_bin: "/openstack/venvs/maas-{{ maas_venv_tag }}/bin"
```

Comment:
Did you rule out `"{{ maas_venv }}/bin"` as a possible value of `maas_venv_bin` here? If so, can you explain why?